### DISCUSSION TITLE
Why Am I Getting A Wrong Configuration Error If It Works?

### DISCUSSION THREAD

**Discussion by SoulTech9** [vercel/community#3540] [Help] [answered]
### Summary

I don't understand why I am getting an error all of a sudden.  Everything works the way I need it to, so I'm not sure if this is some kind of bug or not.

![image](https://github.com/vercel/community/assets/112087610/10fab068-0de9-4623-b8c1-445baa413193)
![image](https://github.com/vercel/community/assets/112087610/20ff8f27-871b-4fbc-aead-9f0b27b43811)

### Steps to Reproduce

I don't understand why I am getting an error all of a sudden.  Everything works the way I need it to, so I'm not sure if this is some kind of bug or not.

![image](https://github.com/vercel/community/assets/112087610/10fab068-0de9-4623-b8c1-445baa413193)
![image](https://github.com/vercel/community/assets/112087610/20ff8f27-871b-4fbc-aead-9f0b27b43811)

**Comment by swarnava** [comment]
Hi,

What is your domain name?

**Reply by SoulTech9** [reply]
I do not wish to share the domain at the moment,  because I am working on it, but as I said it works fine (few months now).  My mx, @ records, everything is A-Okay.  That's why I'm confused about receiving an error message today (everything still works, to be clear).

**Reply by amyegan** [reply]
Hey @SoulTech9. Given that the nameservers definitely do not point the domain to a Vercel project, I'm guessing it was connected via A or CNAME records. Can you show us what the "A Record (Recommended)" tab says?

**Comment by SoulTech9** [comment]
![image](https://github.com/vercel/community/assets/112087610/243802ab-f314-4340-b10f-f68e2599b313)

**Reply by amyegan** [reply, marked as answer]
There's the problem. The `76.76.21.21` record points at Vercel, but the other points somewhere else. You should delete the `162.255.119.247` record to avoid unexpected behavior for your site.

**Comment by SoulTech9** [comment]
Yes, but I am pointing to a different server tho... Im using vercel for the files, but I'm using the other server for Gmail... Please explain, I dont understand?

**Reply by SoulTech9** [reply]
I appreciate your help btw

**Comment by swarnava** [comment]
If you are planning to host a website on Vercel, you must point the `A` record to Vercel IP address as instructed in `Domain` page.

You can also choose to host a specific subdomain such as `blog.yourdomain.com` where `yourdomain.com` can be pointed to different server.

Have a good one!

**Comment by SoulTech9** [comment]
Hey, I just double-checked the domain A Record with my provider and it is as you say, I don't know where its getting 162 ... ip address from. 
![image](https://github.com/vercel/community/assets/112087610/fb043fad-d11c-4e84-9cf4-8ba3f269f8a7)

**Reply by SoulTech9** [reply]
@amyegan

**Reply by SoulTech9** [reply]
This makes sense... I mean, how else would the site be working if it wasn't pointed to the vercel server?

**Reply by SoulTech9** [reply]
So this may be a bug on Vercel's end... not sure tho, I'm too new to know.

**Reply by amyegan** [reply]
If you share the domain, we can help you look into it further. Without knowing the domain I can't tell what else might be causing your issue.

**Reply by SoulTech9** [reply]
I ignore up until just a week ago, because it wouldn't renew the SSL uless the issue was addressed. Turns out the admins from namecheap had to manually fix something on their end... This issue is resolved.
